components/bl-draggable-list-component/README.md

```diff
@@ -1,44 +1,83 @@
-<dd>Allows to specify permission to add elements in list in the Backendless Logic.</dd>
-<dt>On Drag Event</dt>
-<dd>Triggered when the user drags list item.</dd>
-<dt>On Delete Event</dt>
-<dd>Triggered when the user delete item from list.</dd>
-<dt>On Edit Event</dt>
-<dd>Triggered when the user edited list item.</dd>
-<dt>On Add Event</dt>
-<dd>Triggered when the user added item to list.</dd>
-</dl>
+This is a component for Backendless [UI-Builder](https://backendless.com/developers/#ui-builder) designer.
+
+This allows the list to be displayed. List items can also be dragged, modified, removed, and added.
+The component is customizable, you can enable or disable actions with it.
+
+<p align="center">
+  <img alt="main thumbnail" height="220" src="./thumbnail.png" width="358"/>
+</p>
+
+## Properties
+
+| Property    | Type       | Default value | Logic              | Data Binding | UI Setting | Description                                                                                                        |
+|-------------|------------|---------------|--------------------|--------------|------------|--------------------------------------------------------------------------------------------------------------------|
+| itemsList   | *text*     |               | Items List Logic   | YES          | NO         | This is a handler for the logic to determine items list. Signature of list item: `{label: string, value: string}`. |
+| title       | *text*     | "List"        | Title Logic        | YES          | YES        | This is a handler for the logic to determine title.                                                                |
+| allowDelete | *checkbox* | `true`        | Allow Delete Logic | YES          | YES        | This is a handler for the logic to determine if the item is allowed to delete.                                     |
+| allowEdit   | *checkbox* | `true`        | Allow Edit Logic   | YES          | YES        | This is a handler for the logic to determine if the item is allowed to edit.                                       |
+| allowAdd    | *checkbox* | `true`        | Allow Add Logic    | YES          | YES        | This is a handler for the logic to determine if the item is allowed to add.                                        |
+
+## Events
+
+| Name            | Triggers                          | Context Blocks                                                                                                                                                          |
+|-----------------|-----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| On Drag Event   | when the user drags a list item   | `Current list: [{label: string, value: string}, {...}, ...], Dragged item: {label: string, value: string}, Previous list: [{label: string, value: string}, {...}, ...]` |
```

Comment:
Current list: `[{label: String, value: String}, {...}, ...]`

Also try <br/> to make each block on next line